glide.yaml

```diff
@@ -2,23 +2,30 @@ package: github.com/kubernetes-incubator/rktlet
 import:
 - package: k8s.io/kubernetes
   flatten: true
+  version: 71af33120a4dd3fe8d1109a343e44e07f8eafade
   subpackages:
   - pkg/kubelet/api/v1alpha1/runtime
   - pkg/kubelet/container
   - pkg/util/exec
 - package: golang.org/x/net/context
   flatten: true
```

Comment:
@s-urbaniak Yes. This fields got removed after I glide update to include github.com/coreos/rkt/lib